dissect/target/plugins/os/windows/tasks/records.py

```diff
@@ -4,20 +4,12 @@
 
 BootTriggerRecord = TargetRecordDescriptor(
     "filesystem/windows/task/trigger/boot_trigger",
```

Comment:
```suggestion
    "filesystem/windows/task/trigger/boot",
```
Maybe we can rename this to be a bit nicer.